scripts/utils/generate_ts.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+. $(dirname $0)/commons.sh
+
+print N "This script generates a 'ts' file for the location of the Mozilla VPN client"
+print N ""
+
+cd $(dirname $0)/../.. || die
+
+printn Y "Caching the dep scripts... "
+mkdir -p cache || die
+cp scripts/utils/generate_strings.py cache || die
```

Comment:
Are you caching this to work around the fact that the script is different between main and branches, or am I missing the actual reason? If that's the case, it would make more sense to have the same logic that we have in the [l10n workflow](https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/blob/main/.github/workflows/update.yaml#L55-L75):
- It simplifies the whole $PARAMS definition.
- It's easier to clean up, and the same approach would work if we break things again in the future.

Reply:
Yes that is the point. About the params, we are going to remove them in an existing PR. `-g` and `-t` are basically gone in 2.9 already. See this PR: https://github.com/mozilla-mobile/mozilla-vpn-client/pull/3638
In 2.9 we are also starting the migration to cmake and soon-ish we will remove the `src.pro` file. Having 1 single dummy.pro file seems a good solution to be back-compatible.